### ISSUE TITLE
Notify Features page not updated when published through GC Articles

### ISSUE THREAD

**Issue by andrewleith** [cds-snc/notification-planning#992]
## Describe the bug 

When trying to make updates to the English features page, updates are not being published.

Note that the date on the page doesn't ever seem to update and statically says the following no matter how many times I update it:
![image.png](https://images.zenhubusercontent.com/558ca3104f7a08fbb7f36ec0/0f2b658d-8341-4664-b078-9028920f4bc8)

Additionally, there are "invisible links" (links with no text) sprinkled throughout our markup that I am trying to remove and I am not sure how they got there (this could be a future bug card potentially).

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
SEV-3 Minor

## To Reproduce

1. Go to https://notification.canada.ca/features
2. On the Features page, inspect the page the link titled "Learn about using spreadsheets with GC Notify"
3. Note the HTML just above it: `<a href="https://notification.canada.ca/format"></a>`

## Expected behavior

Page should contain the updates I made

## QA
- [ ] scan the /features page with AXE

Describe overall user/system impact to consider when prioritizing this issue.

If applicable

Impact on Notify users:
- Some pages are not accessible because of this; this could affect users of assistive technologies.

Impact on Recipients:
- Some pages are not accessible because of this; this could affect users of assistive technologies.

Impact on Notify team: 
- Increased support calls

## Screenshots
![image.png](https://images.zenhubusercontent.com/558ca3104f7a08fbb7f36ec0/860f9b57-e4ff-45e2-b0cf-a50164a218c9)

**Comment by andrewleith** [comment]
I don't think the notify team is in any position to fix this but I checked for the issue and the page no longer has it.  I think this is related to the cloudfront cache.  

In scanning the rest of the GCA pages for accessibility issues I noticed another page which I subsequently fixed. I did note that the markup did not update until I forcibly cleared the cloudfront cache though.  I am not sure when/why some pages update right away vs others that don't.

In summary, this caching issue still exists but none of our GCA pages are failing accessibility anymore.

**Comment by jzbahrai** [comment]
@jzbahrai to follow up about the GC Article ticket.

**Comment by andrewleith** [comment]
@jzbahrai to QA

**Comment by jzbahrai** [comment]
I inspected the page and got the same result.
